### PR TITLE
create-tauri-app: add --alpha flag when using cargo command

### PR DESCRIPTION
**Pull request by tkanakamalla**
#### What kind of changes does this PR include?

- Minor fix in the documentation

#### Description

While creating a project using `create-tauri-app`, the commands need the `--alpha` as parameter to get V2 features

### CODE REVIEW

**Comment by FabianLars** [comment]
Hey, thanks for opening this PR! The --alpha flag is also required on the other methods (npm, yarn, etc) so i'd appreciate it if you can add them too :)

**Comment by tkanakamalla** [comment]
Hi @FabianLars thanks for the recommendation. I have made the corresponding change for the yarn command.
I have tried below for npm 
`npm create tauri-app@latest --alpha` 
But this does not seem to be the right way so I did not work. I am not an expert with npm or pnpm so could you please suggest what would be the right way to add `alpha` flag to the npm, pnpm and the rest. I can then push another update with that

**Comment by vasfvitor** [comment]
hey, with npm you need to add an extra `--`
`npm create tauri-app@latest -- --alpha`

and pnpm is just like yarn:
`pnpm create tauri-app --alpha`

but bash and powershell I don't know

**Comment by tkanakamalla** [comment]
Thanks @vasfvitor  . I figured for bash and powershell as well. Updated the commit now for all commands.

File: src/content/docs/guides/create/index.mdx [hunk 1]
```diff
@@ -27,33 +27,33 @@ To get started using `create-tauri-app` run one of the below commands in the fol
 <Tabs>
 <TabItem label="Bash">
 ```sh
-sh <(curl https://create.tauri.app/sh)
+sh <(curl https://create.tauri.app/sh) tauri-app --alpha
```
Comment (FabianLars) [inline src/content/docs/guides/create/index.mdx]:
```suggestion
sh <(curl https://create.tauri.app/sh) --alpha
```

Just this works for me too, can you confirm that it does for you as well?

File: src/content/docs/guides/create/index.mdx [hunk 2]
```diff
@@ -27,33 +27,33 @@ To get started using `create-tauri-app` run one of the below commands in the fol
...
 ```
 </TabItem>
 <TabItem label="PowerShell">
 ```ps
-irm https://create.tauri.app/ps | iex
+$env:CTA_ARGS="--alpha";irm https://create.tauri.app/ps | iex
```
Comment (FabianLars) [inline src/content/docs/guides/create/index.mdx]:
```suggestion
$env:CTA_ARGS="--alpha"; irm https://create.tauri.app/ps | iex
```

**Review by FabianLars — APPROVED** [review]

File: src/content/docs/guides/create/index.mdx (hunk 1 above)
Comment (tkanakamalla) [reply src/content/docs/guides/create/index.mdx]: Yeah right, it works without adding the project name in the args. I don't know what I have missed previously. Thanks for pushing the patch for this

**Pull request merged**